python/README.md

```diff
@@ -1,15 +1,34 @@
-# Python Examples
-Python Pub/Sub Examples for HPE Ezmeral Data Fabric Topics
+# kafka-python Examples
+Python Pub/Sub Examples for HPE Ezmeral Data Fabric Topics. This works with the Apache Kafka Wire level Protocol. 
+This module contains Python producer and consumer programs.These programs accept the following 2 arguments:
+1.Bootstrap server and the port: <DataAccessGateway ip address or hostname: port> 
+2.One kafka topic.
 
-## Install kafka-python: pip3 install kafka-python
+Note: For more details look at 'Running Producer Example' and 'Running Consumer Example' section below.
+
+# About Topic creation: 
+The topic is auto created by default. If the user chooses, cli can also used to create the topic.
+1. Auto Topic Creation: By default the auto topic creation is enabled on the cluster. So, when a producer or conusmer program is run, the specified topic is auto created. 
```

Comment:
Spelling error. "conusmer" => "consumer"

Reply:
Thanks! Will correct it.